### DISCUSSION TITLE
Two screens as widoows?

### DISCUSSION THREAD

**Discussion by adryb** [FreeRDP/FreeRDP#9541] [General]
It is possible to setup freerdp as full screen on multiple screens - yes, this works ok, i can aslo setup one screen as windows with smaller resolution that original screen and i have rdp as widows but i woudl like to have two screens as two windows separate - is this possible?

**Comment by akallabeth** [comment]
not with the current clients.
(well, there is a workaround, you can start a multimonitor `Xephyr` or other `Xserver` and run multimon there)
